content/docs/deployment/csminstallationwizard/src/static/js/tests/ui-functions.test.js

```diff
@@ -72,6 +72,7 @@ const CONSTANTS = {
 	CSM_HELM_V1102: "1.3.2",
```

Comment:
We should remove this if we are removing the support for `CSM_HELM_V1102`.

Reply:
Removed it. Thanks!